src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists/index.tsx

```diff
@@ -293,9 +303,13 @@ const ArtistHeaderTitle = styled.div`
   `};
 `
 
-const ArtistHeaderImg = styled.div<{ src?: string }>`
+const ArtistHeaderImg = styled(Flex)<{ src?: string }>`
   flex: 1;
   background: ${color("purple100")};
```

Comment:
Minor but you could remove the `flex: 1` entirely and inline the `background-color`:

```jsx
<ArtistHeaderImg backgroundColor="purple100">
  ...
</ArtistHeaderImg>
```